### PR TITLE
Added Hindi Translations

### PR DESCRIPTION
**Pull request by Samplayz2007**
haha indian go brrrr 
btw i hope ur hrt therapy goes well Lily anyways.. i hope this helps Adios

### CODE REVIEW

File: clientpanel/hl.php [hunk 1]
```diff
@@ -0,0 +1,158 @@
+﻿<?php
+
+// navigation bar
+$lang['nav']['dashboard'] = "डैशबोर्ड";
+$lang['nav']['coins'] = "सिक्के";
+$lang['nav']['mainpage'] = "मुख्य पृष्ठ";
+$lang['nav']['servers'] = "सर्वर";
+$lang['nav']['createserver'] = "अपना सर्वर बनाएं";
+$lang['nav']['changeram'] = "अपना राम बदलो";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:9]: change राम to रैम

Comment (SaumyaKhobragade) [reply clientpanel/hl.php:9]: And बदलो to बदले

File: clientpanel/hl.php [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+$lang['nav']['alwaysonline'] = "हमेशा ऑनलाइन (कार्य प्रगति पर है)";
+$lang['nav']['soon'] = "जल्दी";
+$lang['nav']['account'] = "खाता प्रबंधन";
+$lang['nav']['logintopanel'] = "कंट्रोल पैनल में लॉग इन करें";
+$lang['nav']['plandetails'] = "सदस्यता विवरण";
+$lang['nav']['billing'] = "भुगतान प्रबंधन";
+
+// notices
+$lang['notices']['premiumdiscount'] = "अपने प्लान को पावर और अल्टीमेट में अपग्रेड करते समय कूपन कोड<b> UPGRADE10</b> के साथ <b>6 महीने</b> के लिए <b>10% छूट</b> प्राप्त करें।";
+$lang['notices']['premiumad'] = "24/7 ऑनलाइन और विज्ञापन-मुक्त वेबसाइट जैसी अतिरिक्त सुविधाओं के साथ अपने सर्वर को शीर्ष पायदान हार्डवेयर पर हमारे फेलिक्स प्रीमियम प्लान के साथ बाजार मूल्य से कम पर होस्ट करें। <a href='https://falixnodes.net/minecraft-server-hosting'>मुझे क्लिक करें</a>!";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:19]: Replace फेलिक्स with फलिक्स dude its falix not felix

File: clientpanel/hl.php [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+$lang['links']['coinshop'] = "Falix कॉइन स्टोर में खरीदारी करें";
+$lang['links']['coinshop_description'] = "आपके Minecraft सर्वर के लिए खरीद संसाधन बढ़ा देता है";
+$lang['links']['coins'] = "Falix क्सिक्के मुफ़्त कमाएँ";
+$lang['links']['coins_description'] = "हमारी आभासी मुद्रा अर्जित करके अपने सर्वर को बूस्ट करें";
+$lang['links']['discordsupport'] = "Discord सहायता";
+$lang['links']['discordsupport_description'] = "अपने प्रश्न हमारे समुदाय से पूछें";
+
+// resources widget
+$lang['resources']['serversbalance'] = "सर्वर स्लॉट बैलेंस";
+$lang['resources']['rambalance'] = "RAM संतुलन";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:52]: संतुलन means having same weights between 2 items in a old-styled weighing machine. Use a different better word instead.

File: clientpanel/hl.php [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+$lang['server']['name'] = "सर्वर का नाम";
+$lang['server']['cpu'] = "CPU";
+$lang['server']['ram'] = "RAM";
+$lang['server']['disk'] = "DISK";
+$lang['server']['database'] = "DATABASE";
+$lang['server']['port'] = "PORTS";
+
+$lang['server']['gotoserver'] = "सर्वर प्रबंधित करें";
+$lang['server']['changecpu'] = "परिवर्तन CPU";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:73]: बदले is the suitable word here not परिवर्तन. All the things you see in google are not the correct ones

**Comment by Samplayz2007** [comment]
thank you so much for ur help mate

File: clientpanel/hl.php [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+$lang['server']['deleteserver'] = "सर्वर हटाएं";
+
+// create server page
+$lang['create']['createserver'] = "सर्वर बनाएँ";
+$lang['create']['servername'] = "सर्वर का नाम";
+$lang['create']['serverram'] = "सर्वर रैम";
+$lang['create']['servergame'] = "Game";
+$lang['create']['allgames'] = "सभी समर्थित खेल";
+$lang['create']['selectgame'] = "खेल का चयन करें";
+$lang['create']['minecrafteula'] = "मुझे स्वीकार है<a href='https://account.mojang.com/documents/minecraft_eula/'>Minecraft अंत उपयोगकर्ता लाइसेंस समझौता (EULA)</a>";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:85]: That doesn't even make sense.

File: clientpanel/hl.php [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+$lang['changeram']['currentram'] = "वर्तमान रैम";
+$lang['changeram']['newram'] = "नई रैम राशि";
+$lang['changeram']['changerambutton'] = "रैम बदलें";
+
+// coins page
+$lang['coins']['earnfalixcoins'] = "Falix सिक्के कमाएँ";
+$lang['coins']['resourcestore'] = "Falix सिक्के दुकान";
+$lang['coins']['task'] = "काम";
+$lang['coins']['reward'] = "पुरस्कार";
+$lang['coins']['earn'] = "कमाना";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:100]: make it कमाए

File: clientpanel/hl.php [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+$lang['coins']['claim'] = "दावा";
+$lang['coins']['invitefriends'] = "अपने दोस्तों को आमंत्रित करो";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:102]: replace the last word with करें

File: clientpanel/hl.php [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+$lang['coins']['joinforrewards'] = "अपने दोस्तों को आमंत्रित करें";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:103]: doesn't make any sense

File: clientpanel/hl.php [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+$lang['coins']['invitereward'] = "1,000 Falix प्रति आमंत्रित मित्र के सिक्के";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:104]: remove के, and move सिक्के from last to in between falix and प्रति

File: clientpanel/hl.php (hunk 3 above)
Comment (Samplayz2007) [reply clientpanel/hl.php:52]: suggestions?

File: clientpanel/hl.php [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+$lang['coins']['joinreward'] = "तक 50 Falix प्रत्येक सर्वर से जुड़े सिक्के";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:105]: dude really added marathi in between hindi words anyways lemme know what sentence are you trying to translate

File: clientpanel/hl.php [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+
+// affiliate page
+$lang['affiliate']['title'] = "एक दोस्त को आमंत्रित करें और प्राप्त करें 1,000 Falix सिक्के";
+$lang['affiliate']['description'] = "अपने दोस्तों को आमंत्रित करें और सेकंड में अपना इनाम प्राप्त करें।";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:109]: do not include सेकंड

File: clientpanel/hl.php [hunk 12]
```diff
@@ -0,0 +1,158 @@
...
+$lang['affiliate']['seconddescription'] = "नीचे दिए गए वेबसाइट लिंक को कॉपी करके अपने दोस्तों को आमंत्रित करें और अपने दोस्तों को भेजें।";
+$lang['affiliate']['footer'] = "दोस्तों को आमंत्रित करने के बाद, आप कर सकते हैं<a href='https://client.falixnodes.net/affiliate/status'>अपने रेफरल की स्थिति देखें </a> या यदि आपके कोई प्रश्न हैं तो हमारे डिस्कॉर्ड सर्वर पर हमसे संपर्क करें।";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:111]: दोस्तों को आमंत्रित करने के बाद, आप <a href='https://client.falixnodes.net/affiliate/status'> पर अपने रेफरल की स्थिति देखें </a> या यदि आपके कोई प्रश्न हैं तो हमारे डिस्कॉर्ड सर्वर पर हमसे संपर्क करें।

File: clientpanel/hl.php [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+
+// activity page
+$lang['activity']['title'] = "गतिविधि";
+$lang['activity']['id'] = "पहचान";
+$lang['activity']['action'] = "ACTION";
+$lang['activity']['description'] = "विवरण";
+$lang['activity']['useragent'] = "उपयोगकर्ता एजेंट";
+$lang['activity']['ip'] = "आईपी ​​पता";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:119]: remove ​​पता

File: clientpanel/hl.php [hunk 14]
```diff
@@ -0,0 +1,158 @@
...
+$lang['activity']['date'] = "तारीख";
+
+// profile page
+$lang['profile']['settings_title'] = "पार्श्वचित्र समायोजन";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:123]: remove पार्श्वचित्र

File: clientpanel/hl.php [hunk 15]
```diff
@@ -0,0 +1,158 @@
...
+$lang['profile']['changepassword_notice'] = "नायब! यदि आप गेम पैनल पासवर्ड बदलने की कोशिश कर रहे हैं, तो दूसरे पासवर्ड प्रॉम्प्ट पर नीचे स्क्रॉल करें।";
+$lang['profile']['oldpassword'] = "पुराना पासवर्ड";
+$lang['profile']['newpassword'] = "नया पासवर्ड";
+$lang['profile']['confirmpassword'] = "पासवर्ड की पुष्टि कीजिये";
+$lang['profile']['updatepassword'] = "पासवर्ड अपडेट करें";
+
+$lang['profile']['discord'] = "Discord";
+$lang['profile']['authorised'] = "के रूप में प्राधिकृत";
+$lang['profile']['linkdiscord'] = "जोड़ना Discord";
+$lang['profile']['unlinkdiscord'] = "उतारना Discord";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:149]: replace उतारना with हटाए

File: clientpanel/hl.php [hunk 16]
```diff
@@ -0,0 +1,158 @@
...
+$lang['profle']['notlinked'] = "आपके पास नहीं है Discord account अपने लिंक करने के लिए, खाते से लिंक किया गया Discord account, अपने को अधिकृत करें Discord account";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:150]: doesn't make any sense

File: clientpanel/hl.php [hunk 17]
```diff
@@ -0,0 +1,158 @@
...
+
+$lang['profile']['dangerzone'] = "खतरा क्षेत्र";
+$lang['profile']['changegamepanelpassword_title'] = "गेम पैनल का पासवर्ड बदलें";
+$lang['profile']['changegamepanelpassword_description'] = "गेम पैनल पासवर्ड बदलने के बाद, पुराना पासवर्ड काम नहीं करेगा।";
+$lang['profile']['changegamepanelpassword_button'] = "पासवर्ड बदलें";
+$lang['profile']['deleteaccount_title'] = "इस खाते को हटा दें";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:156]: include Falix से between को and हटा

File: clientpanel/hl.php [hunk 18]
```diff
@@ -0,0 +1,158 @@
...
+$lang['profile']['deleteaccount_description'] = "एक बार जब आप खाता हटा देते हैं, तो वापस नहीं जा रहा है। कृपया निश्चिंत रहें।";
```
Comment (SaumyaKhobragade) [inline clientpanel/hl.php:157]: replace रहा है with सकते

File: clientpanel/hl.php (hunk 5 above)
Comment (SaumyaKhobragade) [reply clientpanel/hl.php:85]: मुझे <a href='https://account.mojang.com/documents/minecraft_eula/'>Minecraft End User License Agreement (EULA)</a> स्वीकार है|

File: clientpanel/hl.php (hunk 8 above)
Comment (SaumyaKhobragade) [reply clientpanel/hl.php:103]: पुरस्कार के लिए Join करें

File: clientpanel/hl.php (hunk 10 above)
Comment (SaumyaKhobragade) [reply clientpanel/hl.php:105]: हर Server Join करके 50 Falix सिक्के प्राप्त करे

File: clientpanel/hl.php (hunk 16 above)
Comment (SaumyaKhobragade) [reply clientpanel/hl.php:150]: आपका Discord Account यह Falix Account से Linked नही है। Link करने के लिए अपना Discord Account अधिकृत करे।

**Comment by Samplayz2007** [comment]
That should be it. Should be ready for merge

**Comment by Samplayz2007** [comment]
Translated the Login Page :)

**Comment by HellCat46** [comment]
This pull request has been closed as we don't need the translation, which is done using an online tool. Please keep it in mind and don't open another pull request until it is a human translation.

Also, from this point forward, Hindi translation will be handled by our staff members (to be more precise, Casper and me).

**Pull request closed without merging**